imagersite/imagersite/templates/registration/activation_complete.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'imagersite/base.html'%}
```

Comment:
I would add a link here to the login page, makes it easier for a user that is registering